pkg/controller.v1alpha3/experiment/util/status_util.go

```diff
@@ -168,7 +168,7 @@ func UpdateExperimentStatusCondition(collector *ExperimentsCollector, instance *
 		return
 	}
 
-	if (instance.Spec.MaxFailedTrialCount != nil) && (failedTrialsCount >= *instance.Spec.MaxFailedTrialCount) {
+	if (instance.Spec.MaxFailedTrialCount != nil) && (failedTrialsCount > *instance.Spec.MaxFailedTrialCount) {
```

Comment:
Do we need the ()?

Reply:
`()` is no necessary. I can change it if there are some other comments :)